src/ServicePulse.Host/vue/src/components/configuration/HealthCheckNotifications_ConfigureEmail.vue

```diff
@@ -98,7 +103,7 @@ function close() {
               </div>
               <div class="row"></div>
               <div class="form-group" :class="{ 'has-error': !toIsValid }">
-                <label for="to">To address</label>
+                  <label for="to">To address <br />(Separate multiple email address with a comma. E.g. testing@test.com,testing2@test.com)</label>
```

Comment:
```suggestion
                <label for="to">To address <br />(Separate multiple email address with a comma. E.g. testing@test.com,testing2@test.com)</label>
```

Indentation change again